modules/sample-audit-log-message.adoc

```diff
@@ -58,3 +63,20 @@ The log message has the following format:
   }
 }
 ----
+
+The source IP address of the request is displayed in the source fields, which makes it easier for you to investigate audit log requests and identify their origin.
+
+To determine the source IP address of a request, {product-title-short} uses the following:
+
+. `xForwardedFor`: The X-Forwarded-For header.
+. `requestAddr`: The remote address header.
+. `sourceIp`: The IP address of the HTTP request.
+
+[IMPORTANT]
+====
+The determination of the source IP address depends on how you expose Central externally:
+
+* If you expose Central behind a load balancer, for example, if you are running it on  Google Kubernetes Engine (GKE) or Amazon Elastic Kubernetes Service (Amazon EKS) by using the Kubernetes External Load Balancer service type, see link:https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip[Preserving the client source IP].
+* If you expose Central behind an Ingress Controller that forwards requests by using the link:https://datatracker.ietf.org/doc/html/rfc7239#section-5.2[X-Forwarded-For header], you do not need to make any configuration changes.
+* If you expose Central with a TLS passthrough route, you can not determine the source IP address of the client. A cluster-internal IP address is displayed in the source fields as the source IP address of the client.
```

Comment:
In case you use the term _parameters_:
```suggestion
* If you expose Central with a TLS passthrough route, you can not determine the source IP address of the client. A cluster-internal IP address is displayed in the source parameters as the source IP address of the client.
```

Reply:
@dhaus67 Can we go ahead with this suggestion.

Reply:
Yeah, parameters is fine as well.